packages/backend/prisma/schema.prisma

```diff
@@ -111,6 +112,7 @@ model Product {
   views           Int           @default(0)
   createdAt       DateTime      @default(now())
   updatedAt       DateTime      @updatedAt
+  inFavorites  FavoriteProducts[]
```

Comment:
It should be related to User Entity

Reply:
I think no.
https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations